### PR TITLE
feat: Kubernetes 1.29.0-rc.1 and other bumps

### PR DESCRIPTION
**Pull request by smira**
Bump Go modules, final tools and semi-final pkgs.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -7,7 +7,7 @@ replace (
 	github.com/siderolabs/talos/pkg/machinery => ./pkg/machinery
 
 	// github.com/cilium/ebpf@v0.11.0/internal/unix/types_linux.go:41:36: undefined: linux.BPF_F_KPROBE_MULTI_RETURN
-	golang.org/x/sys => golang.org/x/sys v0.13.0
+	// golang.org/x/sys => golang.org/x/sys v0.13.0
```
Comment (frezbo) [inline go.mod:10]: remove?

**Review by frezbo — APPROVED** [review]

**Review by Unix4ever — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (smira) [reply go.mod:10]: oh yeah, I found out that it's not needed, but forgot to remove it completely, thanks!

**Pull request merged**